packages/components/src/components/DistributionsChart/SummaryTable.tsx

```diff
@@ -83,7 +83,7 @@ type SummaryTableProps = {
 export const SummaryTable: FC<SummaryTableProps> = ({ plot, environment }) => {
   const showNames = plot.distributions.some((d) => d.name);
   return (
-    <table className="border border-collapse border-slate-400">
+    <table className="border border-collapse border-slate-200 overflow-hidden rounded-lg">
```

Comment:
`rounded-lg` doesn't do what you intended for it to do:
<img width="138" alt="Captura de pantalla 2023-05-28 a la(s) 18 18 54" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/7155687c-ee69-4981-8b26-d76f9d394f39">

I'm also confused about `overflow-hidden`, it doesn't affect anything.

Reply:
This was a mistake - will remove, sorry.